src/chatdlg.cpp

```diff
@@ -106,8 +106,26 @@ void CChatDlg::OnClearChatHistory()
 void CChatDlg::AddChatText ( QString strChatText )
 {
     // add new text in chat window
-    txvChatWindow->append ( strChatText );
-
-    // notify accessibility plugin that text has changed
-    QAccessible::updateAccessibility ( new QAccessibleValueChangeEvent ( txvChatWindow, strChatText ) );
+	
+    // analyze strChatText to check if hyperlink
+	if (strChatText.contains("https://", Qt::CaseInsensitive) | strChatText.contains("http://", Qt::CaseInsensitive)) 
+	{
+		int indx_http  = strChatText.indexOf("http", 0);
+		int indx_space = strChatText.indexOf(" ", indx_http); 
+		if (indx_space==-1) indx_space=strChatText.length();
+
+		int cutslash = (strChatText.at(indx_space-1)=='/') ? 1:0;		// drop "/" if last character of url text
```

Comment:
Why?  That trailing slash _can_ cause different site behaviour (in badly behaved sites).

Reply:
Hi, the trailing slash (if any) would prevent for instance Firefox to open the site
I noticed that when entering as an hyperlink https://www.google.com/ versus https://www.google.com, this is the reason of cutting this trailing slash. 
This is not a real issue anyway since any invalid url will be rejected by the browser :-)

Reply:
Firefox opens (from your post above) both equally well.  If it's a defect in the QTextBrowser widget's URL handling, that's a different matter.  (And another reason to take care with using it for active links.)

Reply:
Really, just simplify it.  If I enter something as simple as `http://example.com/?key=value`, your slash stripper fails.  I'd dump all that code and just find the index of either "http://" or "https://" through to space and pass that string.